code/modules/shuttle/emergency.dm

```diff
@@ -484,10 +484,10 @@
 				setTimer(SSshuttle.emergency_dock_time)
 				send2adminchat("Server", "The Emergency Shuttle has docked with the station.")
 				priority_announce(
-					text = "[SSshuttle.emergency] has docked with the station. You have [DisplayTimeText(SSshuttle.emergency_dock_time)] to board the emergency shuttle.",
-					title = "Emergency Shuttle Arrival",
+					text = "[SSshuttle.emergency] istasyona yanaştı. Acil durum mekiğine binmek için [DisplayTimeText(SSshuttle.emergency_dock_time)] kadar vaktiniz var.",
```

Comment:
DisplayTimeText "3 minutes" şeklinde çevirecek
bunu önlemek için DisplayLocaleTimeText gibi yeni bir proc yaparsan hoş olur

Reply:
DisplayLocaleTimeText yapıldı temel olarak türkçe süreler.
Return Örn. 5 Gün / 11 Saat / 10 Dakika / 3 Saniye.
![image](https://github.com/psychonaut-station/PsychonautStation/assets/55231188/444fc1ac-2347-43a0-bfcb-74ef30e6d507)